### PR TITLE
[form] Updating getUserInput

### PR DESCRIPTION
**Pull request by GuenoleK**
Adjusting performances thanks to @JabX

### CODE REVIEW

File: src/middlewares/validations.js [hunk 1]
```diff
@@ -293,7 +293,8 @@ export const formatValue = (value, entityPath, fieldName, definitions, domains)
       return value;
     }
     const {formatter = defaultFormatter} = domains[domainName] || {};
-    return formatter(value);
+    const {unformatter = defaultFormatter} = domains[domainName] || {};
+    return formatter !== identity ? formatter(value) : unformatter(value);
```
Comment (JabX) [inline src/middlewares/validations.js]: This looks like a separate change from the main one we've been doing, and it seems pretty suspicious to me.

**Comment by JabX** [comment]
The change is OK but you messed with the formatting of JSDoc comments (not that I particularly care anyway).

File: src/middlewares/validations.js (hunk 1 above)
Comment (GuenoleK) [inline src/middlewares/validations.js]: You are absolutly right. Thanks

**Comment by JabX** [comment]
I am curious to know about the "app breaking bug" you encountered with not defining the fieldHelpers for the first render, but you cannot not trust you on this so 👍 for this PR.

**Comment by GuenoleK** [comment]
@JabX yeah i'll show you, maybe you'll got an idea of the problem :)

**Pull request merged**